CreeDictionary/tests/utils_tests/data/layouts/noun-na-basic.layout

```diff
@@ -0,0 +1,35 @@
+name: "Animate noun basic (plain) paradigm"
+paradigm: "noun-na.paradigm"
+layout:
+  type: "basic"
+tooltips:
+  eng:
+    "my": "1st person singular possessor"
+    "your (one)": "2nd person singular possessor"
+    "his/her": "3rd person singular possessor"
+    "our": "myt person plural exclusive possessor"
+    "your and our": "myt person plural inclusive possessor"
+    "your (all)": "2nd person plural possessor"
+    "their": "3rd person plural possessor"
+    "his/her/their (further)": "Obviative possessor"
+    "One": "Singular"
+    "Many": "Plural"
+    "Further": "Obviative"
+    "In/On": "Locative"
+    "Among": "Distributive"
+    "Smaller/Lesser/Younger": "Diminutive"
+    "Ownership": "Possession"
+--
+| "One"        | =N+A+Sg                    |
+| "Many"       | =N+A+Pl                    |
+| "Further"    | =N+A+Obv                   |
+|              |                            |
+|              | : "Smaller/Lesser/Younger" |
+| "One"        | Der/Dim+N+A+Sg             |
+|              |                            |
+|              | : "Ownership"              |
+|              | : "One"                    |
+| "my"         | =N+A+Px1Sg+Sg              |
+| "your (one)" | =N+A+Px2Sg+Sg              |
+|              | : "Further"                |
+| "his/her"    | =N+A+Px3Sg+Obv             |
```

Comment:
these layout files are fixed to test the functionality of the `ParadigmFiller` code, otherwise the slightest change to the layout files will break the tests.

Reply:
This is good!